srcpkgs/ergo/files/ergo/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec vlogger -t ergo
```

Comment:
Please use the symlink approach instead.

Reply:
Whoops, true 😅